polar-c-api/Cargo.toml

```diff
@@ -11,8 +11,7 @@ bench = false
 
 [dependencies]
 polar-core = { path = "../polar-core", version = "0.7.1" }
-serde = { version = "1.0.116", features = ["derive", "rc"] }
-serde_json = "1.0.58"
+serde_json = "1.0.59"
```

Comment:
were we not using the `derive` + `rc` features in this crate? trying to square this change w/ the change in `polar-core/Cargo.toml`, where we kept `serde` and `serde_json` as explicit deps.

Reply:
Talked about this offline. We weren't using serde at all.